.github/workflows/maven-windows.yml

```diff
@@ -25,10 +25,15 @@ jobs:
           graalvm-version: 20.2.0.java11
       - name: Make staging directory
         run: mkdir staging
-      - name: Build and package as Windows application with Gluon
-        run: mvn client:build client:package
-        env:
-          GRAALVM_HOME: C:\hostedtoolcache\windows\GraalVM\20.2.0-java11\x64
+      - name: Gluon Build
+        uses: gluonhq/gluon-build@v1
+        with:
+          gluon-license: ${{ secrets.GLUON_LICENSE }}
+          target: 'windows'
```

Comment:
use 'desktop' here. that will build for the 'host' client target, which is windows